utils/misc.mli

```diff
@@ -462,6 +462,10 @@ val log2: int -> int
        (** [log2 n] returns [s] such that [n = 1 lsl s]
            if [n] is a power of 2*)
 
+val log2_nativeint: nativeint -> int
+(** [log2_nativeint n] returns [s] such that [n = Nativeint.shift_left 1n s]
+    if [n] is a power of 2*)
```

Comment:
this is quite an under-specification (technically, the doc says nothing about the returned value if the input is not a power of 2, although I'm relatively certain the function is used on such arguments....), but since the other functions have the same style of doc, I guess it's okay-ish ?